frontend/components/nav_bar/nav_bar_main.jsx

```diff
@@ -63,7 +63,7 @@ class NavBarMain extends React.Component {
 
                     {this.state.menuOpen ? dropMenu : null}
                 </nav>
```

Comment:
Feel free to ignore if you're going to address this later, I'm new to `nav` elements, are they okay to be nested?

Reply:
I believe they are? As far as I know (which isn't much), they are just like divs except are better tags for accessibility - [Semantic HTML elements](https://www.w3schools.com/html/html5_semantic_elements.asp)